mongo_connector/oplog_manager.py

```diff
@@ -291,28 +235,20 @@ def _should_skip_entry(self, entry):
             else:
                 return True, False
 
-        # Ignore the collection if it is not included.
-        # It is not possible to have include and exclude at the same time.
-        if (self.oplog_ns_set and not self.oplog_ex_ns_set and not
-                self.dest_mapping_stru.match_set(ns, self.oplog_ns_set)):
-            LOG.debug("OplogThread: Skipping oplog entry: "
-                      "'%s' is not in the namespace set." % (ns,))
-            return True, False
-        if (self.oplog_ex_ns_set and not self.oplog_ns_set and
-                self.dest_mapping_stru.match_set(ns, self.oplog_ex_ns_set)):
-            LOG.debug("OplogThread: Skipping oplog entry: "
-                      "'%s' is in the exclude namespace set." % (ns,))
-            return True, False
-
-        # use namespace mapping if one exists
-        ns = self.dest_mapping_stru.get(ns, ns)
-        if ns is None:
-            LOG.debug("OplogThread: Skipping oplog entry: "
-                      "'%s' is not in the namespace set." % (ns,))
-            return True, False
+        # Rename or filter out namespaces that are ignored keeping
+        # included gridfs namespaces.
+        new_ns = self.dest_mapping_stru.map_namespace(ns)
+        if new_ns is None:
+            if is_gridfs_file:
+                # Gridfs files should not be skipped
```

Comment:
I chose not to add the gridfs file set to DestMapping yet so it doesn't support wildcards. I don't have a good reason for why.